bakery/README.md

```diff
@@ -62,6 +62,11 @@ To build the PDF:
 
 `bakery run build-pdf col30939 -d ./data/prealgebra-2e/`
 
+### New Git-Storage Compatible Tasks
+Git-storage compatible tasks are not ready for use outside of the CE Tech yet.
+The following graph represents the (almost) current state of git-storage compatible tasks. Intermediate states are shown in blue, tasks in yellow.
+![Flow for git-storage compatible pipelines](./docs/pipelines.png)
```

Comment:
Still wrapping my head around it 😅 , but very useful visual to have in the docs!

Is it worth adding a note here on how to edit / update in the future? (e.g. I think you used yEd based on the Slack thread, but not sure)

Reply:
Yeah, I'll add a note